packages/medusa/src/api/middlewares/transform-query.ts

```diff
@@ -12,12 +12,21 @@ import { FindConfig, QueryConfig, RequestQueryFields } from "../../types/common"
 import { omit } from "lodash"
```

Comment:
praise(this-file): Think the separation of concerns made this super clean - great work!